### PR TITLE
Added a healthcheck url

### PR DESCRIPTION
**Pull request by shivramk**
I've added a simple health check URL needed by our load balancer. All it does it return a 200 ok to indicate that the service is up.

Not sure if this should this live in web/urls.py instead.

### CODE REVIEW

File: src/sentry/conf/urls.py [hunk 1]
```diff
@@ -45,9 +45,15 @@ def handler500(request):
...
+def handler_healthcheck(request):
+    from django.http import HttpResponse
+    return HttpResponse('ok')
+
+
 urlpatterns = patterns('',
     url(r'^admin/', include(admin.site.urls)),
     url(r'^500/', handler500, name='error-500'),
     url(r'^404/', handler404, name='error-400'),
+    url(r'^healthcheck/', handler_healthcheck, name='healthcheck'),
```
Comment (mattrobenolt) [inline src/sentry/conf/urls.py]: Can you name this something like `_health` instead? That way we avoid any more conflicts with potential organization names, etc.

Also, you'll want a trailing `$`.

File: src/sentry/conf/urls.py [hunk 2]
```diff
@@ -45,9 +45,15 @@ def handler500(request):
     return HttpResponseServerError(t.render(Context(context)))
 
 
+def handler_healthcheck(request):
+    from django.http import HttpResponse
```
Comment (mattrobenolt) [inline src/sentry/conf/urls.py]: I'd prefer to just import this at module level.

**Comment by shivramk** [comment]
I've made changes to avoid url conflicts and moved the import the module level

**Comment by dcramer** [comment]
We actually had our own version of this for getsentry. Not sure why we didn't push it downstream.

**Pull request merged**